src/searchModal/scss/searchModal.scss

```diff
@@ -231,7 +231,7 @@ $classTreeVariables: (
     height: 100% !important;
     padding: 0px !important;
     .navi-container {
-        display: flex;
+        display: flex !important;
```

Comment:
**nitpick:** We must avoid using the `!important` modifier. It prevents to properly extend the style. Unless in a very few edge cases, it should be avoided.
I know it is used elsewhere in TAO, but this a bad legacy.
Is there not any way to do without it?

Reply:
I changed the class name so styles are not overwritten by item preview logic